test-lib.sh

```diff
@@ -1088,10 +1088,15 @@ ct_check_testcase_result() {
 # Uses: $TEST_SET - set of test cases to run
 # Uses: $TEST_SUMMARY - variable for storing test results
 # Uses: $IMAGE_NAME - name of the image being tested
+# Uses: $UNSTABLE_TESTS - set of tests, whose result can be ignored
 ct_run_tests_from_testset() {
   local app_name="$1"
+  local is_unstable=0
   for test_case in $TEST_SET; do
     TESTCASE_RESULT=0
+    if [[ " ${UNSTABLE_TESTS[@]} " =~ " ${test_case} " ]]; then
+      is_unstable=1
```

Comment:
We are in a for loop here and there is no way to set `is_unstable` back to zero so I'm afraid that one unstable test would cause all latter tests to be also considered unstable. Or not?

Reply:
Right, sorry, fixed.